bitcoin/build.rs

```diff
@@ -28,4 +28,10 @@ fn main() {
             println!("cargo:rustc-cfg=rust_v_1_{}", activate_version);
         }
     }
+
+    let feature_std = std::env::var_os("CARGO_FEATURE_STD").is_some();
+    let feature_no_std = std::env::var_os("CARGO_FEATURE_NO_STD").is_some();
+    if !feature_std && !feature_no_std {
+        panic!("One of the cargo features `std` or `no-std` or both must be enabled.");
+    }
```

Comment:
This is not needed, we already have it as `compile_error!` in `lib.rs`.

Reply:
Oh sorry I didn't see that this was already fixed. I removed the change to build.rs.